x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/processors/community_id.tsx

```diff
@@ -0,0 +1,298 @@
+        defaultMessage="Field containing the source IP address."
+      />
+    ),
+  },
+  source_port: {
+    type: FIELD_TYPES.TEXT,
+    serializer: from.emptyStringToUndefined,
+    label: i18n.translate('xpack.ingestPipelines.pipelineEditor.communityId.sourcePortLabel', {
+      defaultMessage: 'Source port (optional)',
+    }),
+    helpText: (
+      <FormattedMessage
+        id="xpack.ingestPipelines.pipelineEditor.communityId.sourcePortHelpText"
+        defaultMessage="Field containing the source port."
+      />
+    ),
+  },
+  destination_ip: {
+    type: FIELD_TYPES.TEXT,
+    serializer: from.emptyStringToUndefined,
+    label: i18n.translate('xpack.ingestPipelines.pipelineEditor.communityId.destinationIpLabel', {
+      defaultMessage: 'Destination IP (optional)',
+    }),
+    helpText: (
+      <FormattedMessage
+        id="xpack.ingestPipelines.pipelineEditor.communityId.destinationIpHelpText"
+        defaultMessage="Field containing the destination IP address."
+      />
+    ),
+  },
+  destination_port: {
+    type: FIELD_TYPES.TEXT,
+    serializer: from.emptyStringToUndefined,
+    label: i18n.translate('xpack.ingestPipelines.pipelineEditor.communityId.destinationPortLabel', {
+      defaultMessage: 'Destination port (optional)',
+    }),
+    helpText: (
+      <FormattedMessage
+        id="xpack.ingestPipelines.pipelineEditor.communityId.destinationPortHelpText"
+        defaultMessage="Field containing the destination port."
+      />
+    ),
+  },
+  icmp_type: {
+    type: FIELD_TYPES.TEXT,
+    serializer: from.emptyStringToUndefined,
+    label: i18n.translate('xpack.ingestPipelines.pipelineEditor.communityId.icmpTypeLabel', {
+      defaultMessage: 'ICMP type (optional)',
+    }),
+    helpText: (
+      <FormattedMessage
+        id="xpack.ingestPipelines.pipelineEditor.communityId.icmpTypeHelpText"
+        defaultMessage="Field containing the destination ICMP type."
+      />
+    ),
+  },
+  icmp_code: {
+    type: FIELD_TYPES.TEXT,
+    serializer: from.emptyStringToUndefined,
+    label: i18n.translate('xpack.ingestPipelines.pipelineEditor.communityId.icmpCodeLabel', {
+      defaultMessage: 'ICMP code (optional)',
+    }),
+    helpText: (
+      <FormattedMessage
+        id="xpack.ingestPipelines.pipelineEditor.communityId.icmpCodeHelpText"
+        defaultMessage="Field containing the ICMP code."
+      />
+    ),
+  },
+  iana_number: {
+    type: FIELD_TYPES.TEXT,
+    serializer: from.emptyStringToUndefined,
+    label: i18n.translate('xpack.ingestPipelines.pipelineEditor.communityId.ianaLabel', {
+      defaultMessage: 'IANA number (optional)',
+    }),
+    helpText: (
+      <FormattedMessage
+        id="xpack.ingestPipelines.pipelineEditor.communityId.ianaNumberHelpText"
+        defaultMessage="Field containing the IANA number. Used only when {field} is not specified."
+        values={{
+          field: <EuiCode>{'transport'}</EuiCode>,
```

Comment:
Do we typically use EUI code formatting to refer to other form fields? I found it a little confusing.

Kibana users won't necessarily know the ES API version of the field. The capitalization also isn't quite the same.

Reply:
We use them mostly to highlight defaults, and I found another processor that also uses this for highlighting another field. We can also highlight them as bold, but feels like it draws too much attention to it:

<img width="553" alt="Screenshot 2021-07-19 at 09 27 27" src="https://user-images.githubusercontent.com/1191206/126112933-82a7846e-35b5-495e-acfa-8eb3f97f1069.png">

Maybe @dborodyansky has some input on this?

Reply:
Thanks for the explanation. Fixing the capitalization helped a lot.  I'll defer to @dborodyansky on proper formatting.